src/types/PlaidTypes.res

```diff
@@ -0,0 +1,257 @@
+type linkLogLevel = DEBUG | INFO | WARN | ERROR
+
+type commonPlaidLinkOptions = {
+  logLevel?: linkLogLevel,
+  extras?: Js.Dict.t<Js.Json.t>,
+}
+
+type linkTokenConfiguration = {
+  token: string,
+  noLoadingState?: bool,
+  ...commonPlaidLinkOptions,
+}
+
+type linkErrorCode =
```

Comment:
We should not maintain an enum for all the different errors that can happen. If Plaid throws any error the SDK should send a log to backend and close plaid. The SDK's course of action will remain same irrespective of any non recoverable error from Plaid SDK. Please remove this.

Reply:
removed these error types 👍